pkg/auth/auth_assisted_service_mock_test.go

```diff
@@ -81,71 +84,81 @@ func (f fakeInventory) DownloadMinimalInitrd(ctx context.Context, params install
 }
 
 func (f fakeInventory) DownloadClusterFiles(ctx context.Context, params installer.DownloadClusterFilesParams) middleware.Responder {
-	file, err := ioutil.TempFile("/tmp", "test.file")
+	file, err := ioutil.TempFile("/tmp", "test")
 	if err != nil {
 		return installer.NewDownloadClusterFilesInternalServerError().WithPayload(
 			common.GenerateError(http.StatusInternalServerError, err))
 	}
+
+	defer os.Remove(fmt.Sprintf("{%s}/{%s}", "/tmp", "test"))
+
 	return filemiddleware.NewResponder(
 		installer.NewDownloadClusterFilesOK().WithPayload(io.ReadCloser(file)),
 		"test",
 		0)
 }
 
 func (f fakeInventory) V2DownloadClusterFiles(ctx context.Context, params installer.V2DownloadClusterFilesParams) middleware.Responder {
-	file, err := ioutil.TempFile("/tmp", "test.file")
-	if err != nil {
-		return installer.NewV2DownloadClusterFilesInternalServerError().WithPayload(
-			common.GenerateError(http.StatusInternalServerError, err))
-	}
 	return filemiddleware.NewResponder(
-		installer.NewV2DownloadClusterFilesOK().WithPayload(io.ReadCloser(file)),
-		"test",
-		0)
+		installer.NewV2DownloadClusterFilesOK().WithPayload(ioutil.NopCloser(strings.NewReader("test"))),
```

Comment:
Prefer `io.NopCloser` over `ioutil`. 
From the go std lib documentation:
> As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code.

ref: https://pkg.go.dev/io/ioutil@go1.17.8

Reply:
done